### PR TITLE
Ignore literals for the sake of defining behavior

### PR DESCRIPTION
**Pull request by jez**
### Motivation

This unblocks some changes that we want to make to flatten where there's more
symbols left behind in various places.

### Test plan

See included automated tests.

### CODE REVIEW

File: test/cli/autogen-autoloader/autogen-autoloader.out [hunk 1]
```diff
@@ -166,7 +166,16 @@ Opus::Require.on_autoload('Yabba::Dabba::Jazz::JazBaz')
...
+Opus::Require.autoload_map(Yabba::Dabba::Jazz::JazBaz, {
+  VALUE: "autoloader/Yabba/Dabba/Jazz/JazBaz/VALUE.rb",
+})
+
+--- output/Yabba/Dabba/Jazz/JazBaz/VALUE.rb
+# frozen_string_literal: true
+# typed: true
+
+
+Opus::Require.for_autoload(nil, "test/cli/autogen-autoloader/bar.rb")
```
Comment (DarkDimius) [inline test/cli/autogen-autoloader/autogen-autoloader.out]: the `nil` here worries me

Comment (DarkDimius) [reply test/cli/autogen-autoloader/autogen-autoloader.out]: Could you double-check if it's safe?

**Comment by jez** [comment]
types: http://go/builds/bui_GiYWLs2jxP9T4p
tests: http://go/builds/bui_GiYiAX5bsFM1zV

File: test/cli/autogen-autoloader/autogen-autoloader.out (hunk 1 above)
Comment (jez) [reply test/cli/autogen-autoloader/autogen-autoloader.out]: There are thousands of lines that look similar in the `/build` folder.

**Comment by jez** [comment]
From conversation with nroman: he was using `'x'` as a cheap way to write test classes that defined behavior. I'm going to change these tests to use a method call instead (`p 'x'`), which should preserve the original intent of the tests.

**Pull request merged**